phpBB/styles/prosilver/template/smiley_dropdown.html

```diff
@@ -0,0 +1,41 @@
+<div id="smiley-dropdown" class="dropdown dropdown-extended" >
+    <div class="pointer"><div class="pointer-inner"></div></div>
+    <div class="smiley-content">
+        <div id="smiley-box" class="smiley-box">
+            <ul>
+            <!-- EVENT posting_editor_smilies_before -->
+            <!-- IF S_SMILIES_ALLOWED and .smiley -->
+
+            <!-- BEGIN smiley -->
+                <li class="smiley-list">
+            <a href="#" onclick="insert_text('{smiley.A_SMILEY_CODE}', true); return false;"><img src="{smiley.SMILEY_IMG}" width="{smiley.SMILEY_WIDTH}" height="{smiley.SMILEY_HEIGHT}" alt="{smiley.SMILEY_CODE}" title="{smiley.SMILEY_DESC}" /></a>
+            </li>
+                    <!-- END smiley -->
+            <!-- ENDIF -->
+            <!-- IF S_SHOW_SMILEY_LINK and S_SMILIES_ALLOWED -->
+            <br /><li><a href="{U_MORE_SMILIES}" onclick="popup(this.href, 750, 350, '_phpbbsmilies'); return false;">{L_MORE_SMILIES}</a></li>
```

Comment:
you are overhauling or replacing the code. It makes sense to overall the whole thing cleanly rather than to perpetuate and old practices/stds. So why not use data attributes to store template vars and jquery to extract and execute the events rather than inline JS.